module/actor/actor-sheet.js

```diff
@@ -332,7 +332,7 @@ export class VampireActorSheet extends ActorSheet {
       data.skill = ''
     }
     if (type === 'boon') {
-      data.boontype = "Trivial"
+      data.boontype = 'Trivial'
```

Comment:
Should probably use a default value that draws from localization but I'm cool without :)